app/classifier/tasks/text/index.cjsx

```diff
@@ -93,3 +83,43 @@ module.exports = React.createClass
       value = textBefore + startTag + textInBetween + endTag + textAfter
     newAnnotation = Object.assign @props.annotation, {value}
     @props.onChange newAnnotation
+
+  handleClear: (e) ->
```

Comment:
I don't think this works as you intended. This'll fire on every keydown, but it's immediately followed up by the `onChange` handler, so the results are never applied.